rust/lance-index/Cargo.toml

```diff
@@ -28,6 +28,7 @@ datafusion.workspace = true
 deepsize.workspace = true
 futures.workspace = true
 half.workspace = true
+hex.workspace = true
```

Comment:
I probably missed it but is this actually used?

Reply:
this is used to encode the protobuf bytes cause now we store the centroids in metadata (which can be only `String`)